cmd/run.go

```diff
@@ -410,6 +424,8 @@ func init() {
 	runCmd.Flags().AddFlagSet(optionFlagSet())
 	runCmd.Flags().AddFlagSet(configFlagSet())
 	runCmd.Flags().StringVarP(&runType, "type", "t", runType, "override file `type`, \"js\" or \"archive\"")
+	runCmd.Flags().BoolVar(&runNoSetup, "no-setup", runNoSetup, "don't run setup()")
+	runCmd.Flags().BoolVar(&runNoTeardown, "no-teardown", runNoTeardown, "don't run teardown()")
```

Comment:
At least for cloud execution we also need to be able to run **only** the `setup` function (and pause) and then distribute the result/return of the `setup` function to all nodes' `default` function.